test/test_pipeline/components/dummy_components/dummy_component_2.py

```diff
@@ -8,9 +8,8 @@
 sys.path.append(parent_directory)
 
 from autosklearn.pipeline.components.base import AutoSklearnClassificationAlgorithm
-from dummy_components.dummy_component_1 import DummyComponent1
 
 
-class DummyComponent2(DummyComponent1):
+class DummyComponent2(AutoSklearnClassificationAlgorithm):
```

Comment:
Although not stated explicitly, this was supposed to check that also inherited components are added. Did this not work any more?

Reply:
I spend a lot of time on debugging this test, locally it was fine, but sunning on Travis it somehow assumed `dummy_component_1.py` to be system module and so it was not listed in further `find_componets` method...
if you have any other proposal how to fix it, it will be welcome :]

Reply:
Thanks for investigating this. Unfortunately, I don't have the time to check this any further and will come back to this in ~2 weeks. I'll merge for now.